install.sh

```diff
@@ -322,7 +300,7 @@ done
 
 # Work around https://github.com/chromebrew/chromebrew/issues/3305.
 # shellcheck disable=SC2024
-# sudo ldconfig &> /tmp/crew_ldconfig || true
+sudo ldconfig &> /tmp/crew_ldconfig || true
```

Comment:
Is this workaround still necessary? We can remove it if it is no longer needed.

Reply:
I'm not sure...